21.md

```diff
@@ -20,9 +32,7 @@ If `SERVICE` wishes for a `WALLET` user to specify a payment amount in a differe
    "minSendable": number,
    "metadata": string,
 +  "currency": {
-+    "code": "PHP",
-+    "name": "Philippine Pesos",
-+    "symbol": "₱",
```

Comment:
I'd keep the `code`, `name, and `symbol` fields as some UIs might have the ability to use whichever is most convenient for them.

Reply:
Nah, actually only having `name` is great! Perfect way to have a preferred currency that may not have a ticker like shamackles.